app/models/miq_report/seeding.rb

```diff
@@ -68,14 +68,15 @@ def sync_from_file(filename, dir, typ)
       rpt[:template_type] = typ
       rec = self.find_by_filename(rpt[:filename])
 
+      type_titleized = typ.titleize
```

Comment:
> In the existing code conversion happens already just once (if/else). You're adding an extra (unused) conversion in case rec.file_mtime.utc >= rpt[:file_mtime]

it the current code it happens every time the if is entered which is per report, that is currently (worst case) > 120 times. I agree that in ideal case I am adding one redundant conversion, but given the alternative...